test/e2e/specs/fakerp/admin.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/openshift/openshift-azure/test/sanity"
 )
 
-var _ = Describe("Openshift on Azure admin e2e tests [Fake]", func() {
+var _ = Describe("Openshift on Azure admin e2e tests [Fake][EveryPR]", func() {
```

Comment:
I'm presuming you're doing this with a view to making EveryPR the focus of the `make e2e` target - as it stands this change is insufficient to make this test run every PR.